porn.txt

```diff
@@ -346731,6 +346734,7 @@
 0.0.0.0 pornoroute.com
 0.0.0.0 pornorscorn.com
 0.0.0.0 pornorubka.net
+0.0.0.0 pornoruhe.net
```

Comment:
```
The page isn’t redirecting properly

An error occurred during a connection to www.pornoruhe.net.

    This problem can sometimes be caused by disabling or refusing to accept cookies.
```

Reply:
@spirillen I have no idea what you want me to change. Please add

Reply:
This is not necessary a request for change, it is a indication that I (me @spirillen ) is unable to confirm this is a porn domain, as the server do not redirect probable (miss configured) when I visited it. 

So this is more like a info + request for other to try confirm this as porn domain